### PR TITLE
Api keystone region name bug

### PR DESCRIPTION
**Pull request by zhangjianweibj**
fix bug
https://github.com/monasca/monasca-docker/issues/494

### CODE REVIEW

File: monasca-api-python/api-config.conf.j2 [hunk 1]
```diff
@@ -122,6 +122,7 @@ password = {{ KEYSTONE_ADMIN_PASSWORD }}
 user_domain_name = Default
 project_name = {{ KEYSTONE_ADMIN_TENANT }}
 project_domain_name = Default
+region_name = {{  KEYSTONE_REGION_NAME }}
```
[the remark this answers is not in the document]
Comment (zhangjianweibj) [reply monasca-api-python/api-config.conf.j2]: ok,very thanks

**Comment by matrixik** [comment]
I would propose to add conditional like is done here:
https://github.com/monasca/monasca-docker/blob/master/monasca-api-python/api-config.conf.j2#L112-L114

If KEYSTONE_REGION_NAME is defined then add set it, if not don't add line with region.

**Comment by zhangjianweibj** [comment]
yeah,very thanks.i will fix this problem.

**Comment by matrixik** [comment]
Fix in #501

**Pull request closed without merging**